src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -100,4 +102,48 @@ install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ./perf_
   WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin/)")
 install(CODE "message(\"-- Created symlink: perf_client -> ./perf_analyzer\")")
 
+# Configure doctest
+include(ExternalProject)
+find_package(Git REQUIRED)
+
+ExternalProject_Add(
+    doctest
+    PREFIX ${CMAKE_BINARY_DIR}/doctest
+    GIT_REPOSITORY https://github.com/doctest/doctest.git
+    TIMEOUT 10
+    UPDATE_COMMAND ${GIT_EXECUTABLE} pull
```

Comment:
why is this needed? Can be removed in my opinion.

Reply:
I just copied and pasted this from here:

https://github.com/doctest/doctest/blob/master/doc/markdown/build-systems.md

I've added the above link as a comment in the code for attribution.

Removed `UPDATE_COMMAND` 👍